policyengine_us/parameters/gov/states/oh/tax/income/credits/joint_filing/other_non_refundable_credits.yaml

```diff
@@ -1,15 +1,10 @@
 description: Ohio provides a joint filing credit equal to a fraction of tax liability less these non-refundable tax credits.
 values:
   2021-01-01:
-    - oh_cdcc # Line 6
-    - oh_senior_citizen_credit # Line 3
-    - oh_retirement_credit # Line 2
-    - oh_exemption_credit # Line 9
-  2023-01-01:
```

Comment:
no changes? did we miss a credit that was either added in 2023 or revoked?

Not sure why we would do this before

Reply:
I don't know how those credits got picked in the list, they are the exact same list comparing 2021 and 2023, so I deleted the 2023 part. We do have a file named "non_refundable_credits.yaml", that file has the adopted credit got revoke in 2023.

Reply:
Makes sense, and in the forms it is all the same?

Reply:
There are 4 new credits that we have not yet programmed: credit for the beginning farmers financial management program, welcome home Ohio credit, Ohio low-income housing credit, affordable single-family housing credit.  ([line 21,22,32,33](https://dam.assets.ohio.gov/image/upload/tax.ohio.gov/forms/ohio_individual/individual/2023/1040-bundle-original.pdf#page=7))

Reply:
What are the details of:
Ohio low-income housing credit, affordable single-family housing credit?

The other two are very low priority